src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs

```diff
@@ -308,6 +311,17 @@ internal virtual void AddCustomSuggestionsForValuePossibilities()
         /// </summary>
         internal virtual void AddCustomSuggestionsForGlobals()
         {
+            foreach (var global in _powerFxConfig.GetSymbols())
+            {
+                var type = DType.Unknown;
+
+                if (global is IExternalOptionSet optionSet)
+                {
+                    type = optionSet.Type;
+                }
```

Comment:
Can symbols know their type? 
Seems like every symbol would need that, and having to cast to specific symbol types is brittle.

Reply:
#287